benchmark/t-test.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+// Class for doing statistic test following a Student's t-distribution,
+// with default configuration: mu = 0, alternative = two.sided
+function TTest(leftData, rightData) {
+  this.left = this.createData(leftData);
+  this.right = this.createData(rightData);
+  this.df = this.left.size + this.right.size - 2;
+  this.dfhalf = this.df / 2;
+  this.mean = this.left.mean - this.right.mean;
+  this.commonVariance = ((this.left.size - 1) * this.left.variance +
```

Comment:
Welch t-test is more applicable when the samples have different sizes. In our case, we always have the same size, so we can assume Student's test as good enough with less computation time. In my experience for samples with the same sample size, Student's test is very reliable... but that's ok, I can implement Welch.